cmd/gardener-controller-manager/app/gardener_controller_manager.go

```diff
@@ -352,7 +356,7 @@ func (g *Gardener) Run(ctx context.Context, cancel context.CancelFunc) error {
 		g.LeaderElection.Callbacks = leaderelection.LeaderCallbacks{
 			OnStartedLeading: func(_ context.Context) {
 				g.Logger.Info("Acquired leadership, starting controllers.")
-				run(ctx)
+				utilruntime.Must(run(ctx))
```

Comment:
Any specific reason why you panic on error here and not log the error just like in the gardenlet?

Reply:
No specific reason, I changed it to log.